source/gui/settingsDialogs.py

```diff
@@ -3517,6 +3521,18 @@ def onSave(self):
 			cs["port"] = int(self.port.GetValue())
 		cs["key"] = self.key.GetValue()
 		self.config["ui"]["play_sounds"] = self.playSounds.GetValue()
+		enabled = self.enableRemote.GetValue()
+		oldEnabled = self.config["enabled"]
+		self.config["enabled"] = enabled
+		if enabled != oldEnabled:
+			import remoteClient
+
+			if enabled and not remoteClient.remoteRunning():
+				log.debug("Initializing Remote")
+				remoteClient.initialize()
+			elif not enabled and remoteClient.remoteRunning():
+				log.debug("Terminating remote")
+				remoteClient.terminate()
```

Comment:
considering debug messages have been added to initialize and terminate, I think these can be removed

```suggestion
				remoteClient.initialize()
			elif not enabled and remoteClient.remoteRunning():
				remoteClient.terminate()
```